doc/changes/changes_1.0.2.md

```diff
@@ -0,0 +1,41 @@
+# Exasol AWS Glue Connector 1.0.2, released 2022-10-26
+
+Code name: Dependency updates on top of 1.0.1
+
+## Summary
+
+The following vulnerabilities in dependencies were fixed by updating the dependencies:
+
+* CVE-2022-42003: Uncontrolled Resource Consumption in `jackson-databind`
+
+## Known Remaining Spark 3.3.4 Vulnerabilities and Sonatype Warnings
```

Comment:
Should we remove this paragraph since we did not update the Spark version to 3.3.4?

Reply:
@morazow: Could you please add a precise explanation, why we cannot upgrade to the latest Spark at the moment and what the consequences are?